docs/architecture/modernize-desktop/toc.yml

```diff
@@ -0,0 +1,5 @@
+- name: Modernizing desktop apps on Windows 10 with .NET Core 3
+  href: index.md
+  items:
+  - name: Introduction
```

Comment:
I thought that the first article after the cover was https://github.com/dotnet/docs/blob/desktop-ebook/docs/architecture/modernize-desktop/why-modern-desktop-applications.md, no?

Reply:
I was just adding the placeholder. I will change the link to "why-modern..."